ui/src/engine/diamond.rs

```diff
@@ -1063,6 +1064,19 @@ impl EventHandler for Runner<DIAMOND_RING_LED_COUNT, DIAMOND_CENTER_LED_COUNT> {
                 );
             }
             Event::VoiceOpenEyes => {}
+
+            Event::CriticalState {
+                state: CriticalState::WifiModuleNotInitialized,
+            } => {
+                self.set_ring(
+                    LEVEL_FOREGROUND,
+                    animations::Static::<DIAMOND_RING_LED_COUNT>::new(
+                        Argb::DIAMOND_FULL_PURPLE,
+                        Some(5.0),
```

Comment:
should the color lasts only 5 seconds? 🤔 what if it's missed by the operator?
you can pass None for infinite time